src/components/ServerGridDashboard/ServerGridDashboard.tsx

```diff
@@ -69,7 +69,7 @@ const gridColumns: Array<GridColumn> = [{
     dataMember: 'CPUData',
     width: 100,
     minWidth: GRID_CELL_MIN_WIDTH,
-    cellFormatter: (cellData) => { return <div className={GetClassForStatus('', cellData.status) + ' server-dashboard-grid-cell-content'} > {cellData.usage ? cellData.usage + '%' : '--'}</div>; },
+    cellFormatter: (cellData) => { return <div className={GetClassForStatus('', cellData.status) + ' server-dashboard-grid-cell-content'} > {cellData.usage != null ? cellData.usage + '%' : '--'}</div>; },
```

Comment:
What if the cellData.usage is undefined or empty?

Reply:
If cellData.usage is undefined the cellData.usage != null will return true, that case is correct.
usage is a number converted to string, it shouldn't be possible that it's empty. 
Is this check necessary?

Reply:
You have to format to '--' if there is no usage for the specific cell, so I think the check is required.